frontend/components/AddSite/AddRepoSiteForm.jsx

```diff
@@ -47,14 +46,7 @@ export const AddRepoSiteForm = ({
   <form onSubmit={handleSubmit}>
     <div className="form-group">
       <GitHubRepoUrlField
-        label="GitHub Repository URL"
-        help={
-          <span>
-            Paste your repository&apos;s GitHub URL here.
-            <br />
-            For example: https://github.com/18f/federalist-docs
-          </span>
-        }
+        label="Already have a GitHub repo for your site? Paste the URL here."
```

Comment:
In error-state, the field+label kinda jump a little down the page:

![error-jump](https://user-images.githubusercontent.com/697848/35820730-26d89448-0a6c-11e8-9257-844ba8e27802.gif)

Reply:
Fixed the jump here 🐸